src/api-guides/vat-reclaim-integration.md

```diff
@@ -0,0 +1,112 @@
+* **Shared Technical Services Videos** at [http://www.concurtraining.com/prdeployment/sts](http://www.concurtraining.com/prdeployment/sts) - The videos found here will help you understand the unique types of transactions that can occur in the Expense product and their resulting impacts on the financial picture in the extract file.
+
+## <a name="integration"></a>Integration
+
+### <a name="quick-connect"></a>Quick Connect
+
+[Quick Connect Scope for Enterprise Apps](./quick-connect-scope-for-enterprise-apps.html)
+
+**Quick Connect** describes the process customers use to connect their Concur site with an App Center Partner's Enterprise application. See the separate Quick Connect scope document for details to guide you through the development of this required piece to your certified application.
+
+### <a name="profile"></a>Company Profile
+
+[Profile Company API v1](/api-reference/profile-beta/company.html#get)
+
+### <a name="obtain-expense-report-data-for-vat-reclaim"></a>Obtain Expense Report Data for VAT Reclaim
+
+Concur customers' Expense and Invoice sites are typically scheduled to close out their approval workflow on a nightly basis (some close out on a weekly basis or other intervals and some close on an on-demand interval).  When the expense or invoice period is closed out within the Concur system, the process results in a compiled list of the expense reports or invoices that have been final-approved since the last time the period was closed.  This process changes the status to "Extracted".  At this point, those expense reports or invoices can no longer be edited by the User, Approver, or Administrator.
+  
+note: Some customers will use the Concur ACH electronic payment service or import a file that changes the status to "confirmed".  In these customers, the same payment status field is changed, so the Partner needs to run the search query twice, once for each payment status described below. i.e. it is possible that you could experience an issue when attempting to get expense reports with an "Extracted" status.  In the process of re-trying that request, the bank(s) involved in the ACH Service (or the client's own confirmation file) could at the same time update the status to "Payment Confirmed".  Therefore, the Partner should query for both Paid (aka as Extracted) and Payment Confirmed to ensure no reports were missed.  Concur is addressing this issue in our next generation of APIs.  At that time, there will be 2 distinct fields to capture the "extracted" status and the "payment confirmed" status i.e. the Extracted status will not be overwritten by the Payment Confirmed status and you will be able to make one call for the desired status at a point in time.
+
+There are 3 steps the Partner's app will take to obtain expense report data:
+
+1. GET a list of Expense Reports.
```

Comment:
Ordered lists with line breaks get wonky. Let's make these items...

```
#### Step 1: GET a list of Expense Reports
```

...and so forth.